### ISSUE TITLE
Collection render to json and lost root key

### ISSUE THREAD

**Issue by smoothdvd** [rails-api/active_model_serializers#1838]
#### Steps to reproduce
*(e.g., detailed walkthrough, runnable script, example application)*

```
#
module Api::V1
  #
  class ArticlesController < ApiController
    # GET /api/v1/articles
    def index
      render json: Articles.all
    end
  end
end
```

```
class ArticleSerializer < ActiveModel::Serializer
  attributes :id, :title
  belongs_to :video
end
```
and output json is:

```
[
  {id: 1,...},
  {id: 2, ...} 
]

```
#### Environment

ActiveModelSerializers Version *(commit ref if not on tag)*: 
0.10.2
Output of `ruby -e "puts RUBY_DESCRIPTION"`: 
2.2.3
OS Type & Version: 
OS X 10.11.6 Beta
Integrated application and version *(e.g., Rails, Grape, etc)*: 
rails-api and rails 5.0

#### Backtrace
*(e.g., provide any applicable backtraces from your application)*

#### Additonal helpful information
*(e.g., Gemfile.lock, configurations, PR containing a failing test, git bisect results)*

**Comment by bf4** [comment]
Looks like you need to set the default adapter to `:json`?

**Comment by zts-rbecker** [comment]
I am seeing this problem as well, but I can absolutely confirm I am setting the adapter to :json.

In my case, rails 4.2.5.1, AMS version 0.10.2, 
ruby -e "puts RUBY_DESCRIPTION": ruby 2.2.1p85 (2015-02-26 revision 49769) [x86_64-linux]

I created a file "config/initializers/ams.rb".  I verified it's getting executed by adding a log statement inside it.  

I've tried all of the following in the initializer:
ActiveModel::Serializer.config.adapter = ActiveModelSerializers::Adapter::Json
ActiveModel::Serializer.config.adapter = :json
ActiveModelSerializers.config.adapter = :json

Behavior is identical as described above;  both collections and individual records are rendered as JSON with no root key.
